force-app/main/default/classes/tasks/BasicTasks.cls

```diff
@@ -23,9 +31,22 @@ public class BasicTasks {
   // a: any integer; n: n >= 0 (n can be greater or equal to 0)
   public static Decimal mathPower(Integer a, Integer n) {
     //write your code below this line
-    Decimal result = 0;
+    Decimal result;
+    
+    if (n == 0) {
```

Comment:
Can you try implementing the algorithm without if-else?

Reply:
Updated.